base/workflow/R/run.write.configs.R

```diff
@@ -22,10 +22,14 @@
 #' @export
 #'
 #' @author David LeBauer, Shawn Serbin, Ryan Kelly, Mike Dietze
-run.write.configs <- function(settings, write = TRUE, ens.sample.method = "uniform", 
+run.write.configs <- function(settings,input_design,write = TRUE,  
```

Comment:
Nit:  Tidyverse style calls for a space after every comma
```suggestion
run.write.configs <- function(settings, input_design, write = TRUE,  
```